boards.txt

```diff
@@ -442,6 +442,12 @@ BLENano.menu.softdevice.s130.upload.maximum_size=151552
 BLENano.menu.softdevice.s130.build.extra_flags=-DNRF51 -DS130 -DNRF51_S130
 BLENano.menu.softdevice.s130.build.ldscript=armgcc_s130_nrf51822_{build.chip}.ld
 
+BLENano.menu.lfclk.lfxo=Crystal Oscillator
```

Comment:
I was thinking maybe some people will want to use the RC instead, if they use these as dev boards, perhaps the final circuit will not have a crystal.